docs/animation/implementing.md

```diff
@@ -8,21 +8,21 @@ you are animating.
 Blocks
 --------
 
-Animations for blocks are done with the `AnimationTESR`, which is a `FastTESR`. Because of this, having a `TileEntity` for your block
+Animations for blocks are done with the `TileEntityRendererAnimation`, which is a `TileEntityRendererFast`. Because of this, having a `TileEntity` for your block
```

Comment:
Last time I checked, the animation API does not work as of 1.15 or was scheduled for removal. Hence, if you check the sidebar, I have that commented out since the docs pages would not be relevant anymore.